eagles-food-backend/DOCUMENTATION.md

```diff
@@ -526,6 +527,7 @@ Organizations are like banks, they hold users and allow them to transfer and red
       "role": "Admin"
     }
   ],
+  }
```

Comment:
oh there's also a "others" section thats just like org, but for people in the other organisations

it wont show up in your normal testing cos everyone is in the same org here, but just duplicate the org list here and add it under